lib/gutenberg/gutenberg-editor-component.js

```diff
@@ -248,6 +248,59 @@ export default class GutenbergEditorComponent extends AsyncBaseContainer {
 		await driverHelper.clickWhenClickable( this.driver, By.css( '.dashicons-no-alt' ) );
 	}
 
+	async insertPaymentButtonDetails( {
+		title = 'Button',
+		description = 'Description',
+		price = '1.00',
+		currency = 'AUD',
+		allowQuantity = true,
+		email = 'test@wordpress.com',
+	} = {} ) {
+		await driverHelper.isElementPresent(
+			this.driver,
+			By.css( '.wp-block-jetpack-simple-payments' )
+		);
+		await driverHelper.setWhenSettable(
+			this.driver,
+			By.css( '.simple-payments__field-title .components-text-control__input' ),
+			title
+		);
+		await driverHelper.setWhenSettable(
+			this.driver,
+			By.css( '.simple-payments__field-content .components-textarea-control__input' ),
+			description
+		);
+		await driverHelper.clickWhenClickable(
+			this.driver,
+			By.css(
+				`.simple-payments__field-currency .components-select-control__input option[value="${ currency }"]`
+			)
+		);
+		await driverHelper.setWhenSettable(
+			this.driver,
+			By.css( '.simple-payments__field-price .components-text-control__input' ),
+			price
+		);
+		if ( allowQuantity === true ) {
+			await driverHelper.clickWhenClickable(
+				this.driver,
+				By.css( '.components-form-toggle__input' )
```

Comment:
I think we should be a bit more specific on thits one - something like `.simple-payments__field-multiple .components-form-toggle__input`